neofetch

```diff
@@ -1209,8 +1233,9 @@ get_uptime() {
                     seconds="${seconds/.*}"
                 ;;
 
-                "AIX" | "IRIX")
-                    t="$(LC_ALL=POSIX ps -o etime= -p 1)"
+                "AIX" | "IRIX" | "HP-UX")
+                    [[ "$os" != "HP-UX" ]] && t="$(LC_ALL=POSIX ps -o etime= -p 1)"
+                    [[ "$os" == "HP-UX" ]] && t="$(LC_ALL=POSIX UNIX95=1 ps -o etime= -p 1)"
```

Comment:
It should be safe to use `UNIX95=1` on non-HP-UX (AIX/IRIX) systems since the variable will be just ignored.

Reply:
Condensed it, thanks!